src/PalindromeNumber/Solution.cpp

```diff
@@ -0,0 +1,78 @@
+// Solution by Imtiaz Ahmed (Github: tiazahmd)
+
+#include <iostream>
+#include <vector>
+
+#include "../Test.h"
+
+using namespace std;
+using namespace leetcode;
+
+class Solution {
+public:
+  string isPalindrome(int x) {
```

Comment:
Why return `string` not `bool`?

Reply:
I started with `bool`, then changed it to `string` when I was testing with `Test.h`. No particular reason for changing it. Would you prefer `bool` or `string`?

Reply:
I think `bool` better.